integration-cli/docker_cli_stack_test.go

```diff
@@ -59,13 +59,13 @@ func (s *DockerSwarmSuite) TestStackDeployComposeFile(c *check.C) {
 
 	out, err = d.Cmd("stack", "ls")
 	c.Assert(err, checker.IsNil)
-	c.Assert(out, check.Equals, "NAME        SERVICES\n"+"testdeploy  2\n")
+	c.Assert(out, check.Equals, "NAME                SERVICES\n"+"testdeploy          2\n")
```

Comment:
I had to change it because the tabwriter in [`formatter/formatter.go`](https://github.com/docker/docker/blob/master/cli/command/formatter/formatter.go#L74) is different from the one that was used in [`stack/list.go`](https://github.com/docker/docker/blob/master/cli/command/stack/list.go#L64).

Yep, that sounds like a good idea! I will do that 👍